ml-testing-toolkit/Chart.yaml

```diff
@@ -36,8 +36,8 @@ dependencies:
     version: 11.1.0
     repository: "file://./chart-connection-manager-backend"
     condition: connection-manager-backend.enabled
-  - name: percona-xtradb-cluster
-    version: 1.0.8
+  - name: mysql
+    version: 9.4.1
     repository: https://charts.helm.sh/stable
```

Comment:
If you are switching charts, please make sure you updated the "repository", otherwise the "update-charts-dep.sh" will not be able to resolve the chart.

Reply:
I would actually suggest undoing this change completely, the reason being that the "values.yaml" configs would need to be updated to also reflect this "chart" change.

This should be handled by the https://github.com/mojaloop/project/issues/3091 story. I have added it to the acceptance criteria.